package.json

```diff
@@ -11,20 +11,20 @@
     "aws-xray-sdk": "^3.0.0"
   },
   "devDependencies": {
+    "@octokit/rest": "^18.2.0",
+    "@types/aws-lambda": "^8.10.63",
     "aws-sdk-mock": "^5.1.0",
     "dotenv": "^8.2.0",
-    "eslint": "^6.8.0",
+    "eslint": "^7.20.0",
     "eslint-config-airbnb-base": "^14.1.0",
-    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-prettier": "^7.2.0",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-jest": "^24.1.0",
-    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-prettier": "^3.3.1",
     "jest": "^26.0.1",
-    "prettier": "^2.0.5",
+    "prettier": "^2.2.1",
```

Comment:
Is there a reason the lockfile in this project is ignored?  (Also ... npm vs yarn ... but we won't get into that!)

I think it's pretty heavily suggested that you lock linters to an exact version, to prevent churn between different users.

Reply:
In the past there's been no reason to lock any of the packages in there, this does seem like a good time to start though. I'll add it.